Features/AdminComponentPreview/README.md

```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Add a full width layout screenshot of a component to its component folder to show a preview in the Wordpress back end and front end.
+
+Components previewed in multiple scenarios:
```

Comment:
Sorry, last adjustments:

"Add layout screenshots of a component to its component folder to show a preview in the front end and WordPress back end.

Components can then be previewed in multiple scenarios:"